src/main/java/mate/academy/springintro/controller/BookController.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.springintro.controller;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.springintro.model.Book;
+import mate.academy.springintro.repository.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
```

Comment:
```suggestion
@RequestMapping("/books")
```

add `server.servlet.context-path=/api` to application.properties